### PR TITLE
refine the conditions for sending the Mattermost notification

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

File: .github/workflows/mattermost-ziti-webhook.yml [hunk 1]
```diff
@@ -13,13 +13,14 @@ on:
...
     name: POST Webhook
     steps:
-    - uses: openziti/ziti-webhook-action@v1
+    - uses: openziti/ziti-webhook-action@v2
       env:
-        ZITI_MATTERMOST_IDENTITY: ${{ secrets.ZITI_MATTERMOST_IDENTITY }}
-      if: ${{ github.actor!= 'dependabot[bot]' && github.repository_owner == 'openziti' && github.event_name != 'pull_request' && env.ZITI_MATTERMOST_IDENTITY != null }}
+        ZITI_MATTERMOST_WEBHOOK_URL: ${{ secrets.ZITI_MATTERMOST_WEBHOOK_URL }}
+      # conditional filters out noise from the bot and running in forks when the necessary secrets are not configured
+      if: ${{ github.actor!= 'dependabot[bot]' && env.ZITI_MATTERMOST_WEBHOOK_URL != null }}
```
Comment (smilindave26) [inline .github/workflows/mattermost-ziti-webhook.yml:23]: Should missing env.ZITI_MATTERMOST_WEBHOOK_URL just cause an error (rather than silently skip)?

**Review by smilindave26 — APPROVED** [review]

File: .github/workflows/mattermost-ziti-webhook.yml (hunk 1 above)
Comment (smilindave26) [reply .github/workflows/mattermost-ziti-webhook.yml:23]: ahh - I see.  you are filtering out noise when running from a fork...

Comment (qrkourier) [reply .github/workflows/mattermost-ziti-webhook.yml:23]: @smilindave26 Right, and so for them error would only be an annoyance. The webhook action should fail noisily if any of the three required secrets are missing.

Comment (qrkourier) [reply .github/workflows/mattermost-ziti-webhook.yml:23]: ...if not skipped, I meant to say.

**Pull request merged**